app/Filters/Conditions/GreaterThan.php

```diff
@@ -11,6 +11,6 @@ class GreaterThan implements ConditionContract
 {
     public function __invoke(Arrayable $item, ConditionalsContract $condition): bool
     {
-        return Arr::get($item->toArray(), $condition->getComparatorField()) > $condition->getComparatorValue();
+        return floatval(Arr::get($item->toArray(), $condition->getComparatorField())) > floatval($condition->getComparatorValue());
```

Comment:
Something I'm not totally 100% on is if these values will always be floats like we might want. It's possible someone may want to use some other kind of thing here.